fem/bilinearform.cpp

```diff
@@ -1275,6 +1275,22 @@ void BilinearForm::Update(FiniteElementSpace *nfes)
    height = width = fes->GetVSize();
 
    if (ext) { ext->Update(); }
+   for (int k = 0; k < domain_integs.Size(); ++k)
+   {
+      domain_integs[k]->Update();
+   }
```

Comment:
Is it time to consider changing how loops like these are expressed?
```suggestion
   for (auto integral : domain_integs) {
      integral->Update();
   }
```